From dennis@stosberg.net  Thu Nov 16 14:12:13 2006
Date: Mon, 13 Nov 2006 09:15:20 +0100
From: Dennis Stosberg <dennis@stosberg.net>
To: Greg Kroah-Hartman <gregkh@suse.de>,
	"Ed L. Cashin" <ecashin@coraid.com>
Subject: aoe: Add forgotten NULL at end of attribute list in aoeblk.c
Message-ID: <20061113081520.G77d5ed8a@leonov.stosberg.net>
MIME-Version: 1.0
Content-Type: text/plain; charset=us-ascii
Content-Disposition: inline

This caused the system to stall when the aoe module was loaded.  The
error was introduced in commit 4ca5224f3ea4779054d96e885ca9b3980801ce13

Signed-off-by: Dennis Stosberg <dennis@stosberg.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/block/aoe/aoeblk.c |    1 +
 1 file changed, 1 insertion(+)

--- gregkh-2.6.orig/drivers/block/aoe/aoeblk.c
+++ gregkh-2.6/drivers/block/aoe/aoeblk.c
@@ -68,6 +68,7 @@ static struct attribute *aoe_attrs[] = {
 	&disk_attr_mac.attr,
 	&disk_attr_netif.attr,
 	&disk_attr_fwver.attr,
+	NULL
 };
 
 static const struct attribute_group attr_group = {
